lightly_solution.ipynb

```diff
@@ -170,20 +150,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 11,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "Starting server, listening at 'localhost:3456'\n",
-      "Serving files in './dataset_clothing_images' and './lightly_runs'\n"
+      "Serving files in '/GitHub/lightly-solution-all-in-one-notebook/dataset_clothing_images' and '/GitHub/lightly-solution-all-in-one-notebook/lightly'\n"
```

Comment:
Unrelated but we should consider always printing the absolute path in lightly-serve.